addons/base_import_module/models/ir_module.py

```diff
@@ -37,7 +37,7 @@ def _import_module(self, module, path, force=False):
         known_mods_names = {m.name: m for m in known_mods}
         installed_mods = [m.name for m in known_mods if m.state == 'installed']
 
-        terp = load_information_from_description_file(module, mod_path=path)
+        terp = load_manifest(module, mod_path=path)
```

Comment:
The module import system unpacks the module in a temporary location, imports the data, then removes the module.

Seems like this could interact quite dangerously with `load_manifest` now caching information forever.

Reply:
Resolved by the fact that `load_manifest` is now the uncached version, and `get_manifest` is the preferred, cached version.